run_scream.syrah_ne30_standard.csh=a_run_link

```diff
@@ -0,0 +1 @@
+/p/lustre2/beydoun1/syrah.default.test_prescribed_CCN_6.ne30_ne30
```

Comment:
This link and the file `run_scream.syrah_ne30_standard.csh` comes from running the example sub script and shouldn't be in the PR.  I think you can just delete them.  Be sure to write `git rm` to remove them.

Reply:
removed.